app/assets/javascripts/student_profile/pdf/GenerateReportGraph.js

```diff
@@ -0,0 +1,59 @@
+import Highcharts from 'highcharts';
+
+export default function generateReportGraph(containerSelector, yAxisLabel, xAxisSettings, title, dataSeries){
+
+  const stacking = (dataSeries.length > 1)
+    ? "normal"
+    : "";
+
+  $(containerSelector).highcharts({
+    chart: {
+      type: 'column'
+    },
+    title: {
+      text: title
+    },
+    credits: false,
+    exporting: {
+      enabled: false
+    },
+    plotOptions: {
+      // This sets charts to render immediately, so we can
+      // synchronously set window.status and pass control back
+      // to wkhtmltopdf to render the PDF.
+      series: { animation: false },
+      column: {
+        stacking: stacking,
+        dataLabels: {
+          enabled: true,
+          color: (Highcharts.theme && Highcharts.theme.dataLabelsColor) || 'white',
+          formatter: function(){
+            const val = this.y < 1 ? '' : this.y;
+            return val;
+          },
+        }
+      }
+    },
+    xAxis: xAxisSettings,
+    yAxis: {
+      title: {
+        text: yAxisLabel
+      },
+      stackLabels: {
+        enabled: true,
+        style: {
+          fontWeight: 'bold',
+          color: (Highcharts.theme && Highcharts.theme.textColor) || 'gray'
+        },
+        formatter: function(){
+          const val = this.y < 1 ? '' : this.y;
```

Comment:
@gabivoicu Can you double check this one? I think we want `this.total` here instead of `this.y`.

Reply:
Can't see my response on the front page so adding a link to it [here](https://github.com/studentinsights/studentinsights/pull/1374/files#diff-9970c0bc40ac1427e00ff25192eb7832L36)